change/@ni-nimble-components-d62c8a19-26e8-448d-831b-ddc56969c9bb.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Created render queue for wafer map component input changes",
+  "packageName": "@ni/nimble-components",
+  "email": "33986780+munteannatan@users.noreply.github.com",
```

Comment:
This change file looks like it might be coming from a different PR (it uses Natan's email and describes #905). Could you try deleting it and re-runnning `npm run change` with a description that more accurately describes this PR?

Reply:
Might be caused by the merge I did earlier. The change file is now up to date with the correct data.